docs/integrations/destinations/bigquery.md

```diff
@@ -213,6 +213,7 @@ tutorials:
 
 | Version     | Date       | Pull Request                                               | Subject                                                                                                                                                                           |
 |:------------|:-----------|:-----------------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 3.0.1 | 2025-06-04 | [61345](https://github.com/airbytehq/airbyte/pull/61345) | Better CSV validation for GCS |
```

Comment:
can you remove this + the metadata.yaml version bump? IMO we should be releasing the entire bigquery PR as 3.0.0